### PR TITLE
Add an icon to the build

### PR DESCRIPTION
**Pull request by ArchEzekiel**
Several points
A) my version of Visual Studios is 2 years old, so its possible that when I opened the build, it changed some of the settings in the CS Project, meaning that it may not build properly  (see: line 5 of YAFC.csproj, which I do not understand)
B) I don't know how I would test to see if it works on MacOS or Linux 
C) If there are any criticisms on the icon itself, I'm willing to work on it, its just what I had come up with 
D) I am not familiar with application development at all, so like, sorry if I followed the wrong tutorial online for making this work
E) I am ALSO still unfamiliar with using GitHub to make Pull Requests on public projects, so, uh, sorry if I did something wrong?

### CODE REVIEW

File: YAFC/YAFC.csproj [hunk 1]
```diff
@@ -2,12 +2,13 @@
   <PropertyGroup>
     <LangVersion>8</LangVersion>
     <OutputType>WinExe</OutputType>
-    <TargetFramework>netcoreapp6.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```
Comment (SWeini) [inline YAFC/YAFC.csproj:5]: this is definitely more correct
however, it's still the incorrect netcoreapp6.0 in the other csproj files, and it should be upgraded to 8.0 anyways

try to keep PRs small and focused and do not put multiple changes into a single PR when possible

**Review by SWeini — CHANGES_REQUESTED** [review]
please revert the changes that are not required or not related, other than that it's fine

File: YAFC/YAFC.csproj (hunk 1 above)
Comment (veger) [reply YAFC/YAFC.csproj:5]: >  should be upgraded to 8.0 anyways

Someone else already made a ticket for this: #25

Comment (veger) [reply YAFC/YAFC.csproj:5]: I would say to leave the original, and we'll upgrade to 8.0 when we work on that ticket. So this one stays clean and only focuses on adding an icon only.

**Comment by veger** [comment]
I can test for Linux. But Linux does not have icons for executable, so I doubt there is anything to see/notice.

File: YAFC/YAFC.csproj (hunk 1 above)
Comment (ArchEzekiel) [reply YAFC/YAFC.csproj:5]: Understood

**Comment by ArchEzekiel** [comment]
> I did a quick search and found [SDL_SetWindowIcon()](https://wiki.libsdl.org/SDL2/SDL_SetWindowIcon), so we could laod the icon into a SDL_Surface and set it as the window icon in YAFCui/Core/WindowMain.cs.
> 
> IF you feel uncomfortable doing so, we can also make a new issue to add this, and leave this PR to add the icon to the application executable only.

Its safe to say that I feel uncomfortable doing so, and I probably need to do more work in setting up my workspace anyways before making more additions to the project, so that I don't get weird changes like changing the TargetFramework from netcoreapp6.0 to net6.0

**Review by SWeini — APPROVED** [review]

**Comment by have-fun-was-taken** [comment]
Thank you guys for your contribution!

**Pull request merged**